plugins/autons.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: autons
+spec:
+  version: v0.2.2
+  homepage: https://github.com/ragrag/kubectl-autons
+  shortDescription: Automatic namespace detection for kubectl commands
+  description: |
+    Automatically detect namespaces for resources in kubectl commands and adding the --namespace flag
+  caveats: |
+    * In cases where a resource is found in multiple namespaces, an error is expected and you will have to specify the desired namespace manually
```

Comment:
This is not so much of a caveat about the plugin itself IMO, you can omit it and it's safe behavior already.